install

```diff
@@ -60,14 +66,29 @@ mkdir -p $FORGE_DIR
 echo "$FORGEUP" > $BINARY
 chmod +x $BINARY
 
-# Source the profile to make sure we have an up-to-date PATH
-source $HOME/.profile
+# Store the correct profile file (i.e. .profile for bash or .zshrc for ZSH)
+case $SHELL in
+*/zsh) 
+    echo -e ${GREEN}Detected your preferred shell is ${RED}zsh${GREEN}, adding ${RED}forgeup${GREEN} to ${RED}.zshrc${GREEN}. Please run ${RED}source \~/.zshrc${GREEN} after this script runs, or start a new terminal session to use ${RED}forgeup${GREEN}.${NC}
+    PROFILE=$HOME/.zshrc
+    ;;
+*/bash)
+    echo -e ${GREEN}Detected your preferred shell is ${RED}bash${GREEN}, adding ${RED}forgeup${GREEN} to ${RED}.profile${GREEN}.
+    PROFILE=$HOME/.profile
+    source $PROFILE
+    ;;
+*)
+    echo -e ${RED}Error: ${GREEN}could not detect shell, please manually add ${RED}$FORGE_DIR${GREEN} to your ${RED}PATH${GREEN} variable!${NC}
+    exit 1
+esac
 
 # Only add the forgeup directory to profile if it isn't in path
 if [[ ":$PATH:" != *":$HOME/.forgeup:"* ]]; then
     # Add the forgeup directory to the path and source it
-    echo "export PATH=$PATH:$FORGE_DIR" >> $HOME/.profile
+    echo "export PATH=$PATH:$FORGE_DIR" >> $PROFILE
 fi
 
-# Source the updated profile
-source $HOME/.profile
\ No newline at end of file
+# Source the updated profile if we are in bash
+if [[ "$SHELL" == *"bash"* ]]; then
```

Comment:
can you add and else statement for sourcing zshrc

Reply:
I tried that actually, but the script runs in bash because it starts with `#!/usr/bin/env bash`. So, if you try to source the `.zshrc` file then, it actually throws an error.  We obviously shouldn't change it to run in zsh as it's not as ubiquitous.

I'm not sure if there is a better way to reload the `.zshrc` from within a script which needs to work in both bash and zsh. From what I've seen, telling the user to manually reload the shell is quite common, and I guess that's why.

Reply:
FWIW, I just installed rust and even that tells you to reload your shell rather than trying to do it automatically.

<img width="494" alt="Screenshot 2022-01-15 at 19 49 34" src="https://user-images.githubusercontent.com/89424366/149634248-a6add74a-4206-4d2b-b912-894e7a2f426f.png">

Reply:
alright we should do that unilaterally as well then for consistency imo

Reply:
Cool, agreed. Pushed a commit which should address that nicely now.